packages/connector-local/src/connector.ts

```diff
@@ -5,6 +5,29 @@
  * It currently only sends `fetch::end::*` events.
  */
 
+/**
+ * `jsdom` always tries to load `canvas` even though it is not needed for
+ * the HTML parser. If there is a mismatch between the user's node version
+ * and where the HTML parser is being executed (e.g.: VS Code extension)
+ * `canvas` will fail to load and crash the excution. To avoid
+ * that we hijack `require`'s cache and set an empty `Module` for `canvas`
+ * and `canvas-prebuilt` so `jsdom` doesn't use it and continues executing
```

Comment:
Nit: Can you remove the part about `canvas-prebuilt`? We don't do that anymore (and I know this is a copy paste from somewhere that should also be updated). 🙏